.github/workflows/run_examples.yml

```diff
@@ -37,3 +37,12 @@ jobs:
         # make sure to test *installed* version of pyFstat
         pip install -e $GITHUB_WORKSPACE
         (cd .. && python $GITHUB_WORKSPACE/examples/run_all_examples.py)
+        examples/*/Pyfstat_example_data
+    - name: Save output files as artifacts (*.sft files excluded!)
+      uses: actions/upload-artifact@v2
+      with:
+        path: |
+          $GITHUB_WORKSPACE/examples/*/Pyfstat_example_data/*
+          !$GITHUBWORKSPACE/examples/*/Pyfstat_example_data/*/*.sft
```

Comment:
Going by [this failed run](https://github.com/PyFstat/PyFstat/runs/1087472020?check_suite_focus=true), by how the paths are built locally, and the `..` in L39 above (of which I don't really remember why it's there :wink: ): I guess these should just be just `$GITHUB_WORKSPACE/Pyfstat_example_data/*`

Reply:
Just did an `ls $GITHUB_WORKSPACE`:
```
 CHANGELOG.md
Dockerfile
LICENSE
MANIFEST.in
README.md
ephem
examples
pyfstat
requirements.txt
setup.cfg
setup.py
tests.py
versioneer.py
```

I'll try not to use wild cars and upload folders.